docs/build_docs.sh

```diff
@@ -142,6 +148,35 @@ if [[ -d $docbasepath/build/sphinx ]]; then
     echo ""
 fi
 
+current_release_notes=`ls docs/source/releases/latest/*`
+if [[ "$current_release_notes" != "" ]]; then
+    echo "Running brassy to generate current release note ${geoips_vers}.rst"
+    echo ""
+    echo "touch $docbasepath/docs/source/releases/${geoips_vers}.rst"
+    echo "brassy --release-version $geoips_vers --no-rich \"
+    echo "    --output-file $docbasepath/docs/source/releases/${geoips_vers}.rst \"
+    echo "    $docbasepath/docs/source/releases/latest"
+    echo ""
+
+    touch $docbasepath/docs/source/releases/${geoips_vers}.rst
+    brassy --release-version $geoips_vers --no-rich \
+        --output-file $docbasepath/docs/source/releases/${geoips_vers}.rst \
+        $docbasepath/docs/source/releases/latest
+    if [[ "$?" != "0" ]]; then
+        echo "FAILED brassy ${geoips_vers}.rst release note generation failed."
+        echo "Please resolve release note formatting noted above and retry"
+        exit 1
+    fi
+    echo "Adding latest section to release note index"
+    echo "python $geoipsdocpath/update_release_note_index.py $docbasepath/docs/source/releases/index.rst $geoips_vers"
+    python $geoipsdocpath/update_release_note_index.py $docbasepath/docs/source/releases/index.rst $geoips_vers
+    if [[ "$?" != "0" ]]; then
+        echo "FAILED update_release_note_index.py for version ${geoips_vers}"
+        echo "Please resolve release note formatting noted above and retry"
+        exit 1
+    fi
+fi
```

Comment:
If there are YAML release notes in the latest folder, that means we need to run the stuff that will a) create the release note RST file (either latest.rst or vX_Y_Z.rst) and b) update index.rst to include a link to the new release note.